ReactCommon/fabric/components/text/paragraph/ParagraphShadowNode.cpp

```diff
@@ -38,21 +37,22 @@ void ParagraphShadowNode::setMeasureCache(
   measureCache_ = cache;
 }
 
-void ParagraphShadowNode::updateLocalDataIfNeeded() {
+void ParagraphShadowNode::updateStateDataIfNeeded() {
   ensureUnsealed();
-
+    
   auto attributedString = getAttributedString();
-  auto currentLocalData =
-      std::static_pointer_cast<const ParagraphLocalData>(getLocalData());
-  if (currentLocalData &&
-      currentLocalData->getAttributedString() == attributedString) {
-    return;
-  }
 
-  auto localData = std::make_shared<ParagraphLocalData>();
-  localData->setAttributedString(std::move(attributedString));
-  localData->setTextLayoutManager(textLayoutManager_);
-  setLocalData(localData);
+// Todo: fixme
+//  auto currentLocalData =
+//      std::static_pointer_cast<const ParagraphLocalData>(getLocalData());
+//  if (currentLocalData &&
+//      currentLocalData->getAttributedString() == attributedString) {
+//    return;
+//  }
+
+  auto stateData = getState()->getData();
```

Comment:
That's wrong. You are copying `data` and then modifying it and they throwing it away.

You can change that to `auto &stateData = ...` or simply inline.